pkgs/applications/video/netflix-shortcut/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchurl, google-chrome, lib, makeDesktopItem }:
+
+(makeDesktopItem {
+  name = "Netflix via Google Chrome";
+  exec = "${google-chrome}/bin/${google-chrome.meta.mainProgram} --app=https://netflix.com";
```

Comment:
Nit: this exec= line will bitrot if the desktop file is placed in ~/.config/autostart. An idea could be to write a one-liner script named e.g. `netflix-in-google-chrome`, which could be referenced by name in the desktop file.